src/main/python/utils/cloudsql.py

```diff
@@ -122,17 +122,33 @@ def insert_instance(config):
     cloudsql = discovery.build('sqladmin', 'v1beta4', credentials=credentials)
 
     json_config = json.loads(config)
-    operation = cloudsql.instances().insert(project=json_config["project"], body=json_config).execute()
-    result = wait_for_operation(cloudsql, json_config["project"], operation["name"])
+    instance_config = json_config["databaseInstance"]
+
+    operation = cloudsql.instances().insert(project=instance_config["project"], body=instance_config).execute()
+    result = wait_for_operation(cloudsql, instance_config["project"], operation["name"])
     if "error" in result:
-        raise Exception(result["error"])
-    
+       raise Exception(result["error"])
+
     instanceName = result["targetId"]
-    projectId = result["targetProject"]
+    projectId = result["targetProject"]    
+
+    if "databaseUser" in json_config and json_config["databaseUser"] is not None :
+        add_user(instanceName, instance_config, json_config["databaseUser"])
 
     with open('instance.json', 'w') as instance_file:
         json.dump(cloudsql.instances().get(project=projectId, instance=instanceName).execute(), instance_file, indent=2, sort_keys=True)
 
+def add_user(instanceName, instance_config, user_config):
+    credentials = GoogleCredentials.get_application_default()
+    cloudsql = discovery.build('sqladmin', 'v1beta4', credentials=credentials)
+    projectId = instance_config["project"] 
+
+    operation = cloudsql.users().insert(project=projectId, instance=instanceName, body=user_config).execute()
```

Comment:
@jmesterh @delguan 

We are NOT chopping off the domain name from the iam user while adding a user to an instance, but we do it while querying the db, here- https://github.com/susom/wdl-kit/blob/main/src/main/python/gcp/cloudsql.py#L307 

Should we make it consistent?

While testing, I realized that in theory, iam users(i.e. CLOUD_IAM_USER) other than service accounts can also be added as users to the instance, as long as the user email length is <= 63 characters. [Our pipeline service accounts email (with domain) is > 63 characters, hence we have to add it as CLOUD_IAM_SERVICE_ACCOUNT and leave the domain name out.]

If we think we will never need to add CLOUD_IAM_USER to instance from wdl, then we should chop off the domain name while adding a user to instance too.